rotkehlchen/chain/evm/transactions.py

```diff
@@ -195,11 +195,19 @@ def _query_and_save_transactions_for_range(
         If update_ranges is True, updates the database tracking for this query range.
         Otherwise, data is fetched without updating the query range.
         """
+        period_as_blocks = period
```

Comment:
nitpick: Move this to an else: clause here and below in the similar places to avoid double assignment in the case the if is true.